config/default.schema.json

```diff
@@ -354,6 +354,11 @@
 					],
 					"description": "Value for the default language"
 				},
+				"DEFAULT_TIMEZONE": {
```

Comment:
make this required, add default value in default.json otherwise in https://github.com/hpi-schul-cloud/schulcloud-client/pull/2235/files#diff-c26141a500590e1ee89b521efd4b6d91e4b49e85802f4f038de92271e1b1ffabR5 Configuration.has must be called (which ist not desired).